### PR TITLE
Feature five

### PR DESCRIPTION
**Pull request by Maxwell011**
### Hello in this PR I did the following 👍 

- Final design matches the required design
- Styled the Add book form component.
- Styled the Header component.
- Styled the Booklist component.
- Refactored my reducer code.

Kindly Review 🙏

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,27 +1,135 @@
...
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+.author {
+  color: #4386bf;
+}
```
Comment (cynthiainga) [inline src/index.css]: - [ ] Please kindly make sure your design stick as much as possible to the original design. The size of your author is not the same as in the original design, please kindly fix that 🙃 

Your design

Screen Shot 2022-11-11 at 22 43 02

Original design

Screen Shot 2022-11-11 at 22 43 17

- [ ] Also make sure you use the same font family as in the original design for all the elements in your project 🙃😁

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Maxwell011, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [ ] Pleas kindly check the comment under the review 👇👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Actually, you have 2 `README` files in your repo `ReadME.MD`  and ` README .md` files. I suggest you delete one, the `ReadME.MD` file 😁😁
- The commit message should be written in the imperative tense. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Kindly remember this in your next commits messages 😁🙃

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — COMMENTED** [review]
Hi @Maxwell011 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a README file. It seems that in making the changes requested by the previous reviewer, you totally deleted your README file. Kindly ensure your project has a README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED ✔️ 🟢 

## Hi @Maxwell011 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



## Highlights 🎯 
✔️  Descriptive PR 📌 
✔️ Good commit history 📌 
✔️ No linter errors 📌 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**